vowpalwabbit/cb_explore_adf_greedy.cc

```diff
@@ -90,11 +90,12 @@ VW::LEARNER::base_learner* setup(VW::setup_base_i& stack_builder)
   new_options
       .add(make_option("cb_explore_adf", cb_explore_adf_option)
                .keep()
+               .necessary()
```

Comment:
this works because of line 102 right?

Reply:
I'll need to look closer here.

Reply:
i think it should work but yea i would double check